build.gradle

```diff
@@ -234,6 +234,10 @@ project(':azkaban-webserver') {
     compile('org.mortbay.jetty:jetty:6.1.26')
     compile('org.mortbay.jetty:jetty-util:6.1.26')
 
+    testCompile('junit:junit:4.11')
+    testCompile('org.hamcrest:hamcrest-all:1.3')
+    testCompile('commons-collections:commons-collections:3.2.2')
```

Comment:
Order by alphabetical order -> Done

it appears to be testRuntime dependency -> './gradle distTar' seems to fail unless I have it as testCompile